### DISCUSSION TITLE
トリガーを作ってみました

### DISCUSSION THREAD

**Discussion by chaftta** [kyubuns/AkyuiUnity#26] [Ideas]
AkyuiUnityの勉強を兼ねてトリガーを作ってみました
https://github.com/chaftta/BindableLayoutTrigger

AkyuiUnityは、色々できて夢が広がります！

**Comment by kyubuns** [comment]
ありがとうございます！
AkyuiUnity自体は、「UnityEditor上でuGUIを使ってUIを組む」という作業を置き換えるだけのもので、
そこから先どう制御するかは選択肢が多い方が良いと思っているので、こういう使いやすいトリガーを作ってもらえるのは助かります！
